containers/swift/.devcontainer/devcontainer.json

```diff
@@ -16,9 +16,7 @@
 	],
 
 	// Set *default* container specific settings.json values on container create.
-	"settings": {
-		"lldb.library": "/usr/lib/liblldb.so",
```

Comment:
@damuellen sorry I may have confused things - this is definitely still required - it's the `"vadimcn.vscode-lldb",` line in extensions we don't need